internal/tiltfile/tiltfile_docker_compose_test.go

```diff
@@ -490,6 +490,20 @@ dc_resource('no-svc-with-this-name-eek', 'gcr.io/foo')
 	f.loadErrString("no Docker Compose service found with name")
 }
 
+func TestDockerComposeDoesntSupportEntrypointOverride(t *testing.T) {
```

Comment:
How do you handle invalid entrypoints for dockerBuild/customBuild? For example if a given entry point doesn't exist, how do we handle it?

Reply:
i'm not sure what you mean by "doesn't exist"? but to answer your question, we don't try to validate that here -- we let the container try to run the entrypoint, and report any errors back. trying to validate the entrypoint here would be like trying to validate a user's dockerfile before we ran it -- potennnntially useful but kind of a waste of time.